chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -597,7 +610,19 @@ impl PeerManagerActor {
                     .entered();
 
                     match response {
-                        Ok(routing::actor::Response::AddVerifiedEdgesResponse(filtered_edges)) => {
+                        Ok(routing::actor::Response::AddVerifiedEdgesResponse(
+                            mut filtered_edges,
+                        )) => {
+                            // Don't send tombstones during the initial time.
+                            // Most of the network is created during this time, which results
+                            // in us sending a lot of tombstones to peers.
+                            // Later, the amount of new edges is a lot smaller.
```

Comment:
Interesting! So in a sense the problem here is not necessary tombstones, but rather the fact that we echo back the info we already know. 

Like, a new node connects to 5 peers, the peers send the full graph to the new node, and then the new node broadcasts the whole graph back, but it is kinda useless, as everyone else knows it already. 

With time-based nonces, I think an elegant solution would be to only broadcast edges with timestamp >= node's own startup time. 

What worries me here is that I think the current logic might lead to *new* tombstones not propagating through the network. I am not sure we've seen a retransmit logic anywhere, so I think this means that a fresh tombstone might get lost. But I think this is fine.

Reply:
yes - especially if you're a new node with 'empty database'.

We also had a bug (fixed couple days ago) - where this happened for all the nodes.